src/Engine/Trigger/TriggerRepeat.cpp

```diff
@@ -191,8 +191,8 @@ Json::Value TriggerRepeat::Save() {
         component["triggerFunction"] = targetFunction.front(); // ADD SUPPORT FOR VECTOR
     if (!targetEntity.empty())
         component["triggerTargetEntity"] = targetEntity.front()->GetUniqueIdentifier(); // ADD SUPPORT FOR VECTOR
-    if (!collidedEntity.empty())
-        component["triggerCollidedEntity"] = collidedEntity.front()->GetUniqueIdentifier(); // ADD SUPPORT FOR VECTOR
+    //if (!collidedEntity.empty())
+    //    component["triggerCollidedEntity"] = collidedEntity.front()->GetUniqueIdentifier(); // ADD SUPPORT FOR VECTOR
```

Comment:
These should be uncommented, or a bug will be introduced when updating against master.